### PR TITLE
Doc updates: graph traversal (xg, yg, gqp) improvements

### PR DESCRIPTION
**Pull request by nruppin**

### CODE REVIEW

File: docs/API.Traversal.Operands.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+This query retrieves the outbound links in the **hasImmediateParent** fields of the matched infotons, and in addition, the outbound links *all* fields of the level 1 outbound links.
+
+<a name="hdrFieldValue"></a>
+### Field Value Filter ###
+
+You can add field filters to constrain the additional infotons retrieved by the **xg** flag. You do this by adding the field filters in [ ] brackets after the expansion definition.
+
+    xg=<outboundExpansion>[field filter]
+
+>**Note:** The syntax of the field filter is the same as for the [qp parameter](API.FieldConditionSyntax.md), except that the fuzzy match operator ~ is not supported for **xg** queries.
```
Comment (hochgi) [inline docs/API.Traversal.Operands.md]: > ~ is not supported for **outbound link** queries (xg, as well as yg & gqp)

File: docs/API.Traversal.gqp.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+The **gqp** flag can be used together with **xg** and **yg**. In this case, **gqp** takes precedence, meaning that first results are filtered by the **gqp** expression, and then expanded by **yg** and **xg**.
+
+>**Notes:** 
+>* See [Traversal Operands](API.Traversal.Operands.md) to learn about **gqp** operands.
+>* The [ghost skips](API.Traversal.yg.md#hdrGhostSkips) behavior applies to **qgp** as well as **yg**.
+>* The **gqp** flag can be applied to both **consume** and **search** operations. Note that when using **gqp** with consume, it's possible to filter out the entire chunk and therefore to receive no results for some iterations. If in this case you receive a 204 error, but the position token in the header is different than the one sent, you still need to keep consuming.
```
Comment (hochgi) [inline docs/API.Traversal.gqp.md]: 204 is not an error, it is a legitimate response saying "No Content"

File: docs/API.Traversal.Operands.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+:-------|:------------|:---------------|:--------
+[xg with no operands](#hdrNoOperands) | Adds all level-1 outbound links in all fields | xg | ```<cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership&with-data&xg```
+[xg traversal level](#hdrLevel) | Adds all outbound links in all fields of query results, then all outbound links of the expanded set, and so on up to the traversal level specified. | xg | ```<cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership&with-data&xg=3```
+[xg traversal fields](#hdrXgField) | Adds all outbound links in the specified field or fields. As this option is only available for the **xg** flag, it refers implicitly to outbound links, even though the > operator is not used. | xg | ```<cm-well-host>/permid.org?op=search&qp=CommonName.mdaas:Marriott%20Ownership&with-data&xg=hasImmediateParent.mdaas```
```
[the remark this answers is not in the document]
Comment (nruppin) [reply docs/API.Traversal.Operands.md]: Right, but for that we have the "Multiple inbound or outbound traversal fields" entry in the row below.

[the remark this answers is not in the document]
Comment (hochgi) [reply docs/API.Traversal.Operands.md]: maybe I'll add leading `>` in xg as optional, and same in yg (where's if not provided, implicitly means `>`).
@dudi3001 @YoniMataraso WDYT?

**Pull request merged**